app/routes/notices-setup.routes.js

```diff
@@ -281,7 +281,7 @@ const routes = [
   },
   {
     method: 'GET',
-    path: '/notices/setup/{sessionId}/preview-return-forms',
+    path: '/notices/setup/{sessionId}/preview-return-forms/{returnId}',
```

Comment:
This isn't going to work. Look at the returns tab and the view return log page and you'll see the problem we face trying to use the return id, for example, `v1:6:01/117:10749123:2025-04-01:2026-03-31`.

You'll either need to make this a query param, or pass the value in the body for a post request.